src/bin/inx-chronicle/main.rs

```diff
@@ -146,34 +148,42 @@ async fn main() -> Result<(), Error> {
     Ok(())
 }
 
-fn set_up_logging() {
-    #[cfg(feature = "opentelemetry")]
-    {
-        use tracing_subscriber::prelude::*;
+#[allow(unused)]
```

Comment:
It looks like `set_up_logging` is always used in `main`. Please introduce `allow` statements only with limited scope, i.e. should only put them right in front of the problematic statement/block.